src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,268 @@
+---
+title: Filter panel
+description: The filter panel is a component that can be customized to provide categorized filters to help users trim down large amounts of content, data and/or search results.
```

Comment:
```suggestion
description: The filter panel is a component that can be customized to provide categorized filters to help users trim down large amounts of content, data, and/or search results.
```

Reply:
Adding a comma before the conjunction here to comply with IBM Style, which follows the Chicago Manual of Style serial comma rule. 🐕‍🦺